src/circuit/sapling/mod.rs

```diff
@@ -51,7 +52,11 @@ pub struct Spend<'a, E: JubjubEngine> {
     pub ar: Option<E::Fs>,
 
     /// The authentication path of the commitment in the tree
-    pub auth_path: Vec<Option<(E::Fr, bool)>>
+    pub auth_path: Vec<Option<(E::Fr, bool)>>,
```

Comment:
I think it is better to represent the position as a separate integer, btw (as I argued in https://github.com/zcash/zcash/issues/2831#issuecomment-375983803 ). But that is independent of this PR.